modules/cli/src/main/scala/scala/cli/commands/InstallHome.scala

```diff
@@ -0,0 +1,39 @@
+package scala.cli.commands
+
+import caseapp._
+import coursier.env.{EnvironmentUpdate, ProfileUpdater}
+
+import java.io.File
+import scala.util.Properties
+
+object InstallHome extends ScalaCommand[InstallHomeOptions] {
+  override def hidden = true
+  override def names = List(
+    List("install", "home")
```

Comment:
Let's not add this name here. I believe the default would give `scala-cli install-home` without the override (while it's `scala-cli install home` with the override here, so 3 args instead of 2).

That should allow not to conflict with an `install` command, if we were to add one later on.